README.md

```diff
@@ -93,34 +54,44 @@ docker run -v /path/to/config/dir:/config \
            ghcr.io/maplibre/martin --config /config/config.yaml
 ```
 
-# Usage
+## Usage
 
-Martin requires at least one PostgreSQL [connection string](#postgresql-connection-string) or a [tile source file](#mbtile-and-pmtile-sources) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
+### PostGIS sources
+Martin requires at least one PostgreSQL [connection string](https://maplibre.org/martin/PostgreSQL-Connection-String.html) or a [tile source file](https://maplibre.org/martin/MBTile-and-PMTile-Sources.html) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
 
 ```shell
 martin postgresql://postgres@localhost/db
 ```
 
 Martin provides [TileJSON](https://github.com/mapbox/tilejson-spec) endpoint for each [geospatial-enabled](https://postgis.net/docs/using_postgis_dbmanagement.html#geometry_columns) table in your database.
 
-# API
+### MBTiles and PMTiles sources
```

Comment:
I think at the end of each section we might provide a link to the relevant section of the book?  Or as part of the text?